apps/client/src/app/components/rule/rule-settings-dialog/rule-settings-dialog.html

```diff
@@ -1,76 +1,126 @@
 <div mat-dialog-title>{{ data.rule.name }}</div>
 
 <div class="py-3" mat-dialog-content>
-  <div
-    class="w-100"
-    [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMin }"
-  >
-    <h6 class="mb-0">
-      <ng-container i18n>Threshold Min</ng-container>:
+  @if (
+    data.rule.configuration.thresholdMin && data.rule.configuration.thresholdMax
+  ) {
+    <div
+      class="w-100"
+      [ngClass]="{
+        'd-none':
+          !data.rule.configuration.thresholdMin ||
+          !data.rule.configuration.thresholdMax
+      }"
+    >
+      <h6 class="mb-0">
+        <ng-container i18n>Threshold</ng-container>:
+        @if (data.rule.configuration.threshold.unit === '%') {
+          {{ data.settings.thresholdMin | percent: '1.2-2' }}
+        } @else {
+          {{ data.settings.thresholdMin }}
+        }
+        -
+        @if (data.rule.configuration.threshold.unit === '%') {
+          {{ data.settings.thresholdMax | percent: '1.2-2' }}
+        } @else {
+          {{ data.settings.thresholdMax }}
+        }
+      </h6>
+      @if (data.rule.configuration.threshold.unit === '%') {
+        <label>{{
+          data.rule.configuration.threshold.min | percent: '1.2-2'
+        }}</label>
+      } @else {
+        <label>{{ data.rule.configuration.threshold.min }}</label>
+      }
+      <mat-slider
+        [max]="data.rule.configuration.threshold.max"
+        [min]="data.rule.configuration.threshold.min"
+        [step]="data.rule.configuration.threshold.step"
+      >
+        <input matSliderStartThumb [(ngModel)]="data.settings.thresholdMin" />
+        <input matSliderEndThumb [(ngModel)]="data.settings.thresholdMax" />
+      </mat-slider>
       @if (data.rule.configuration.threshold.unit === '%') {
-        {{ data.settings.thresholdMin | percent: '1.2-2' }}
+        <label>{{
+          data.rule.configuration.threshold.max | percent: '1.2-2'
+        }}</label>
       } @else {
-        {{ data.settings.thresholdMin }}
+        <label>{{ data.rule.configuration.threshold.max }}</label>
       }
-    </h6>
-    @if (data.rule.configuration.threshold.unit === '%') {
-      <label>{{
-        data.rule.configuration.threshold.min | percent: '1.2-2'
-      }}</label>
-    } @else {
-      <label>{{ data.rule.configuration.threshold.min }}</label>
-    }
-    <mat-slider
-      name="thresholdMin"
-      [max]="data.rule.configuration.threshold.max"
-      [min]="data.rule.configuration.threshold.min"
-      [step]="data.rule.configuration.threshold.step"
+    </div>
+  } @else {
+    <div
+      class="w-100"
+      [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMin }"
```

Comment:
This is fine, because you only want to display this slider if `thresholdMin` is set